mindsdb/integrations/handlers/google_docs_handler/README.md

```diff
@@ -0,0 +1,57 @@
+# Google Docs API Integration
+This handler integrates with the Google Docs API to make the docs content available for the use case of NLP model training and predictions.
+
+# Connect to the Google Docs API 
+First we need to create a database to connect to the Google Docs API.
+
+However, you will need to have a Google account and have enabled the Google Docs API. Also, you will need to have a doc id from your google docs and the credentials for that google doc in a json file.
+
```

Comment:
Can we have a link to the google docs for generating the credentials file? If there are some scopes that needs to be setup we should document them too